airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -565,7 +565,7 @@ class GoogleDisplayVideo360SDFtoGCSOperator(BaseOperator):
     :type delegate_to: str
     """
 
-    template_fields = ("operation_name", "bucket_name", "object_name", "body_request")
+    template_fields = ("operation_name", "bucket_name", "object_name")
```

Comment:
Unrelated change?

Reply:
I will rebase, it was fixed in other PR.